wgengine/router/router_windows.go

```diff
@@ -120,23 +147,34 @@ func cleanup(logf logger.Logf, interfaceName string) {
 // See https://github.com/tailscale/tailscale/issues/785.
 // So this tracks the desired state and runs the actual adjusting code asynchrounsly.
 type firewallTweaker struct {
-	logf logger.Logf
-
-	mu          sync.Mutex
-	didProcRule bool
-	running     bool     // doAsyncSet goroutine is running
-	known       bool     // firewall is in known state (in lastVal)
-	want        []string // next value we want, or "" to delete the firewall rule
-	lastVal     []string // last set value, if known
+	logf    logger.Logf
+	tunGUID windows.GUID
+
+	mu             sync.Mutex
+	didProcRule    bool
+	running        bool     // doAsyncSet goroutine is running
+	known          bool     // firewall is in known state (in lastVal)
+	wantLocal      []string // next value we want, or "" to delete the firewall rule
+	lastLocal      []string // last set value, if known
+	wantKillswitch bool
+	lastKillswitch bool
+
+	// Only touched by doAsyncSet, so mu doesn't need to be held.
+
+	// fwProc is a subprocess that runs the wireguard-windows firewall
```

Comment:
document the lifetime of this Cmd? this can come & go & come back based on config, right?